pkgs/by-name/pi/pixelpwnr-server/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, cmake, freetype, libX11
+, libXcursor, libXrandr, libXi, libGL, pkg-config }:
```

Comment:
While I think it is optional for now, it would be nice to format the file with nixfmt-rfc-style

```
nix run nixpkgs#nixfmt-rfc-style -- pkgs/by-name/pi/pixelpwnr/package.nix
```